src/components/dialogs/parameters/common/wrapper-input.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useStyles } from '../parameters';
+import * as yup from 'yup';
+import { FormProvider, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import React, { useEffect } from 'react';
+import { Box, Grid, Typography } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+const WrapperInput = ({ value, label, callback, validator, children }) => {
+    const classes = useStyles();
+
+    const schema = yup.object().shape({
+        value: validator,
+    });
+
+    const methods = useForm({
+        defaultValues: {
+            value: value,
+        },
+        resolver: yupResolver(schema),
+    });
+
+    const { handleSubmit, watch } = methods;
+
+    // to commit by onChange rather than onSubmit
+    useEffect(() => {
+        const subscription = watch(handleSubmit(callback));
```

Comment:
not sure why but this useeffect looks suspicious

Reply:
Similar to other tabs, what do we expect is to commit a value input by user on the fly. When user begin typing, every change is captured then is passed to handleSubmit() which will invoke a callback later if input value is validated (by resolver) 

The solution is proposed following to the discussion at:

https://stackoverflow.com/questions/63466463/how-to-submit-react-form-fields-on-onchange-rather-than-on-submit-using-react-ho